assets/queries/k8s/anonymous_auth_is_not_set_to_false/query.rego

```diff
@@ -0,0 +1,45 @@
+package Cx
+
+import data.generic.k8s as k8sLib
+
+CxPolicy[result] {
+	resource := input.document[i]
+	specInfo := k8sLib.getSpecInfo(resource)
+	container := specInfo.spec.containers[j]
+	commands := ["kube-apiserver", "kubelet"]
+
+	inArray(container.command, commands[_])
+	containerArgs := object.get(container, "args", {})
```

Comment:
```suggestion

```

I think this line does not make sense here since this rule will return a result if `--anonymous-auth=true` is defined in the `container.args`. So, this rule considers that args is defined in the container 


Additionally that, I think `--anonymous-auth=true` can be defined inside `container.command`. For example:
```
apiVersion: v1
kind: Pod
metadata:
  name: command-demo
  labels:
    purpose: demonstrate-command
spec:
   containers:
      - command:
        - kube-apiserver
        - --anonymous-auth=false
        name: kube-apiserver
```